woocommerce/woocommerce-ios
# [Woo POS] UI - Cart: Product rows

Closes #12724 
Closes #12706 

## Description
This PR updates the design for the product rows within the cart view, so they're closer to the initial wireframe. I've used the figma values of `120` for height, `56` for circled buttons (+, x, .. ), and `20`/`32` for padding.

Ref: TfaZ4LUkEwEGrxfnEFzvJj-fi-6_1657

![Simulator Screen Recording - iPad Pro (12 9-inch) (6th generation) - 2024-05-16 at 10 16 34](https://github.com/woocommerce/woocommerce-ios/assets/3812076/075911ea-6d72-42bc-965d-1d896b898384)


## Testing instructions
* In POS mode, observe the changes on:
  * The `Cart` title aligned and using `.title`
  * The product rows 
  * The default-styled "Pay now" button is now a styled-"Checkout" button

Adding and removing products from the cart should behave normally.
